package/gluon-core/luasrc/lib/gluon/upgrade/140-firewall-rules

```diff
@@ -1,25 +1,87 @@
 #!/usr/bin/lua
+-- TODO: replace REJECT by DROP. only use REJECT for debugging
 
 local site = require 'gluon.site_config'
 local uci = require('simple-uci').cursor()
 
+uci:section('firewall', 'zone', 'wan', {
+	name = 'wan',
+	network = { 'wan','wan6',},
+	input = 'REJECT',
+	output = 'ACCEPT',
+	forward = 'REJECT',
+	mtu_fix = '1',
+	conntrack = '1',
+	log = '1',
+}
+)
+-- the client zone is set up by gluon-client-bridge
+--
+uci:section('firewall', 'zone', 'mesh',
+{
+	name = 'mesh',
+	input = 'REJECT',
+	output = 'ACCEPT',
+	forward = 'REJECT',
+	conntrack = '1',
```

Comment:
The conntrack is necessary for respondd I assume? This is quite unfortunate, as it would track all connections going through the node with Babel...